### ISSUE TITLE
KSC's strange twitching in flight mode

### ISSUE THREAD

**Issue by Aebestach** [Kopernicus/Kopernicus#634]
I'm not sure if anyone has reported this issue before, but I did find this error before, so I thought I'd mention it in the issue.

This problem was first discovered in the Kcalbeloh System. When the orbit is in this system, the KSC launch site logo on Kerbin will start to shake violently for no reason.
I think this problem is related to the precision of floating point numbers, and it may be impossible to fix this problem?

bug picture
![image](https://github.com/user-attachments/assets/17112c4b-6ba7-4fbe-b6a7-058cc109a7af)

**Comment by R-T-B** [comment]
Yeah, a floating point precision issue and unfortunately I've done about all I can there to stabilize actual inflight gameplay, but bugs like this remain.  We should leave this open so people know about it, but to be clear, I know of no possible fix at the moment.
